docs/guides/core-concepts/retry-ability.mdx

```diff
@@ -29,25 +29,30 @@ fails, check out our guide on [Test Retries](/guides/guides/test-retries).
 
 ## Commands, Queries and Assertions
 
-While all methods you chain off of `cy` in your Cypress tests are commands,
-there are some different types of commands it's important to understand:
-**queries**, **assertions** and **actions** have special rules about
-retryability. For example, there are 4 queries, an action, and 2 assertions in
-the test below.
+While all methods you chain off of `cy` in your Cypress tests are commands, it's
+important to understand the different rules by which they operate. **Queries**
+link up, retrying the entire chain together, while **non-queries** only execute
+once. Assertions are a type of query that's specially displayed in the command
+log. For example, there are 7 queries (2 of them assertions) and 2 non-queries
```

Comment:
IMO the logic of assertions are in parallel with query commands under the hood, but they act differently, so keeping the concepts separate makes it easier to digest their purposes and overall behaviors.

Reply:
They don't act differently any more; that's the whole point of this work.

Assertions are 100% queries in all ways - they are implemented as them and they behave as them, because they are them.